### ISSUE TITLE
React sometimes crashes upon loadup

### ISSUE THREAD

**Issue by CaptainAmpersand** [VolmitSoftware/React#126]
https://hastebin.com/recuzowupa.md

**Comment by CaptainAmpersand** [comment]
Full log (Which I got a second time trying to enable it, lol) https://hastebin.com/witurecumi.md

**Comment by cyberpwnn** [comment]
Additional logs https://hastebin.com/uhihifusim.md

**Comment by cyberpwnn** [comment]
It looks like with that bug, that the sample controller is failing to start properly. React reloads itself to fix it (which it does fix the issue) but if it cant reload itself, hmmm

**Comment by cyberpwnn** [comment]
#78 #127 also

**Comment by cyberpwnn** [comment]
I may have fixed it in 6.25934a. Needs testing

**Comment by CaptainAmpersand** [comment]
Still not fixed in 6.26120a
[20:30:09 INFO]: Ampersand issued server command: /re reload
[20:30:09 INFO]: [React] Disabling React v6.26120a
[20:30:09 INFO]: Waiting on Surge Cruncher 0
[20:30:09 INFO]: Waiting on Surge Sideline
[20:30:09 INFO]: Waiting on Surge Cruncher 1

**Comment by Rbot48** [comment]
Same issue with PaperSpigot 1.10.2 & React-6.26411a, after running /re reload
```
19.01 13:53:20 [Server] INFO Disabling React v6.26411a
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 0
19.01 13:53:20 [Server] INFO Waiting on Surge Sideline
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 1
```

**Comment by cyberpwnn** [comment]
Waiting on threads is intended. The errors above are not. 
```
19.01 13:53:20 [Server] INFO Disabling React v6.26411a
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 0
19.01 13:53:20 [Server] INFO Waiting on Surge Sideline
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 1
```
That is not an error, but this is
```
[13:14:45 WARN]: [React] Task #1869645 for React v6.24129a generated an exception
java.lang.NullPointerException: null
at react.surge.util.PluginUtil.load(PluginUtil.java:423) ~[?:?]
at react.surge.Main.requestReload(Main.java:133) ~[?:?]
at react.surge.Main.requestReload(Main.java:119) ~[?:?]
at react.controller.SampleController.restartSuperSampler(SampleController.java:73) ~[?:?]
at react.controller.SampleController$2.run(SampleController.java:159) ~[?:?]
at react.surge.sched.TaskLater$1.run(TaskLater.java:36) ~[?:?]
at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.2.jar:git-Spigot-9b8bba4-bd30843]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_152]
```

Is this still an issue?

**Comment by CaptainAmpersand** [comment]
The issue is that's the end of it:
There is no continuation after that. It just Surge Crunches, and stops.
Yes, it is still an issue. :P

**Comment by cyberpwnn** [comment]
So you are saying when you see this 
```
19.01 13:53:20 [Server] INFO Disabling React v6.26411a
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 0
19.01 13:53:20 [Server] INFO Waiting on Surge Sideline
19.01 13:53:20 [Server] INFO Waiting on Surge Cruncher 1
```

The server locks up? Or surge just wont start.

**Comment by Sikatsu** [comment]
After that, the server continues, but react is disabled. Doing /re says unknown command.

**Comment by cyberpwnn** [comment]
Implemented in React 6.0.50
